<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAnalyse</class>
 <widget class="QWidget" name="DataAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据分析</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DataAnalyse{border-image:url(:/images/background2.png)}</string>
  </property>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;固体氧化物电池(SOC)数据分析小工具&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_da">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>数据处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_plot">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>制图</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_da">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gb_da{border: 0.5px solid gray}</string>
   </property>
   <property name="title">
    <string>数据处理</string>
   </property>
   <widget class="QCheckBox" name="cb_dup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据去重</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_vari">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据去偏离值,k=</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsb_vari">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>66</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_del">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据去头尾</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_plot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gb_plot{border: 0.5px solid gray}</string>
   </property>
   <property name="title">
    <string>制图参数</string>
   </property>
   <widget class="QCheckBox" name="cb_av">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电流-电压</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_ap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电流-功率</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_ta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间-电流</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_step">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>取点步长：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_step">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>66</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_upload">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>表格上传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_down">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>表格下载</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="plot">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
